insights/combiners/lspci.py

```diff
@@ -115,15 +115,17 @@ class LsPci(list):
     def __init__(self, lspci_k, lspci_vmmkn):
         if lspci_vmmkn:
             for dev in lspci_vmmkn:
+                dev = dev.copy()
```

Comment:
@xiangce would it be possible to add a comment for the copies to explain why they're needed, to help prevent it from possibly happening again in the future?

Reply:
@ryan-blakley yes, comment should be added. Please have a check again. Thanks.

Reply:
@xiangce Thanks, it looks good to me.